content/en/docs/tasks/administer-cluster/out-of-resource.md

```diff
@@ -27,6 +27,8 @@ The `kubelet` can proactively monitor for and prevent total starvation of a
 compute resource. In those cases, the `kubelet` can reclaim the starved
 resource by proactively failing one or more Pods. When the `kubelet` fails
 a Pod, it terminates all of its containers and transitions its `PodPhase` to `Failed`.
+If the evicated Pod is part of e.g. a Deployment, another Pod will be created and 
```

Comment:
```suggestion
If the evicted Pod is part of e.g. a Deployment, the Deployment will create another Pod to be
```
(spelling, plus using active voice as per website style guide)

What do you think @woosley ?